controllers/accounts.js

```diff
@@ -109,99 +88,92 @@ router.get('/search' , function (req, res, next) {
     const itemsPerPage = 10;
     const currentPage = parseInt(req.query.p) || 1;
 
-    api(req).get('/accounts/', {
+    api(req, { useCallback: false, json: true, version: 'v3' })
+        .get('/account', {
             qs: {
-                username: {
-                    $regex: _.escapeRegExp(req.query.q),
-                    $options: 'i'
-                },
-                $limit: itemsPerPage,
-                $skip: itemsPerPage * (currentPage - 1),
-                $sort: req.query.sort
+                type: 'username',
+                value: req.query.q,
+                limit: itemsPerPage,
+                skip: itemsPerPage * (currentPage - 1),
```

Comment:
The `$sort` property was deleted without replacement.

Reply:
There was no way to change the sort parameter from the UI, so I removed the sort parameter and return a sorted list directly from the server.

Reply:
Is a sort parameter sent from the UI? Then we should remove it as well, to not give the impression it changes anything.